### ISSUE TITLE
Natspec multiple "///" comments not interpreted

### ISSUE THREAD

**Issue by gpersoon** [ethereum/solidity#7150]
## Description

Check the documentation https://solidity.readthedocs.io/en/v0.5.10/natspec-format.html;
it shows to use multiple /// lines to document different values.
However only the last /// lines is incorporated in the compiled natspec data.
A workaround is using /** ... */ comments, then solc behaves as expected.

Please either change the documentation or the compiler; currently it is confusing.

## Environment
```xml
solc --version
solc, the solidity compiler commandline interface
Version: 0.5.10+commit.5a6ea5b1.Windows.msvc
```
Also noticed on Remix

## Steps to Reproduce
Use the following contract "comments.sol" as input:
```solidity
pragma solidity ^0.5.1;
contract Comments {    
    /// @author f1
    function f1() public {}

    /// @author f2
    /// @dev    dev2
    function f2() public {}

    /** @author f3
       @dev    dev3
    */   
    function f3() public {}
}
```
Then compile it using:
```xml
solc comments.sol --devdoc
```

Now you get the following output:
```json
======= comments.sol:Comments =======
Developer Documentation
{
  "methods" : 
  {
    "f1()" : 
    {
      "author" : "f1"
    },
    "f2()" : 
    {
      "details" : "dev2"
    },
    "f3()" : 
    {
      "author" : "f3\r",
      "details" : "dev3\r"
    }
  }
}
```

I would have expected the following output for function f2(). Notice the author line is missing above.
```json
    "f2()" : 
    {
      "author" : "f2\r",
      "details" : "dev2\r"
    }
```

**Comment by leonardoalt** [comment]
Hi @gpersoon , thanks for reporting!
I couldn't reproduce it locally (Arch Linux):
```
16:42 $ ./solc/solc --devdoc natspec.sol 
Warning: This is a pre-release compiler version, please do not use it in production.

======= natspec.sol:Comments =======
Developer Documentation
{
  "methods":
  {
    "f1()":
    {
      "author": "f1"
    },
    "f2()":
    {
      "author": "f2",
      "details": "dev2"
    },
    "f3()":
    {
      "author": "f3",
      "details": "dev3"
    }
  }
}
```
I'll try on other environments as well.
Could you please verify it's not a version issue?

**Comment by leonardoalt** [comment]
Also using your code on Remix gave me exactly the same result.

**Comment by leonardoalt** [comment]
@christianparpart could you please check if it happens on Windows?

**Comment by gpersoon** [comment]
I've checked a few solc versions but it is not version dependent.
I also check Ubuntu and indeed there it works fine.

I've narrowed it down a bit: the problem is with CR+LF in Windows text files.
Problem file: [Comments-with-CRLF.sol.txt](https://github.com/ethereum/solidity/files/3448039/Comments-with-CRLF.sol.txt)
Working fine: [Comments-with-LF.sol.txt](https://github.com/ethereum/solidity/files/3448040/Comments-with-LF.sol.txt)

The problem is probably in liblangutil/Scanner.cpp function scanSingleLineDocComment where a second line with /// is detected, but only if a single LF is present.
